### PR TITLE
march ux update

### PR DESCRIPTION
**Pull request by Blackbaud-ToddRoberts**

### CODE REVIEW

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-ux-update-202003](https://developer.blackbaud.com/skyux?_sp.spa=pr-ux-update-202003)

**Comment by Blackbaud-BobbyEarl** [comment]
Preview build at [https://developer.blackbaud.com/skyux?_sp.spa=pr-ux-update-202003](https://developer.blackbaud.com/skyux?_sp.spa=pr-ux-update-202003)

File: src/app/blog/index.html [hunk 1]
```diff
@@ -5,6 +5,28 @@
...
+      <h3 class="sky-subsection-heading">Recently completed</h3>
+        <h4 class="sky-emphasized"><a href="https://developer.blackbaud.com/skyux-lookup/docs/country-field">Country selector</a></h4>
+					<p>We added a control to allow users to search for and select a country.</p>
+        <h4 class="sky-emphasized"><a href="https://developer.blackbaud.com/skyux-layout/docs/inline-delete">Inline delete confirmation</a></h4>
+        <p>We added inline delete functionality to <a href="https://developer.blackbaud.com/skyux/components/repeater">repeaters</a> and <a href="https://developer.blackbaud.com/skyux/components/grid">grids</a>.</p>
+        <h4 class="sky-emphasized"><a href="https://developer.blackbaud.com/skyux-layout/docs/back-to-top">Back to top</a></h4>
+					<p>We added a bar with a back to top button that allows user to quickly jump back to the top of long lists.</p>
+		</section>
+			<h3 class="sky-subsection-heading">In the works</h3>
+        <h4 class="sky-emphasized"><a href="https://github.com/orgs/blackbaud/projects/3">Theming</a></h4>
```
Comment (blackbaud-johnly) [inline src/app/blog/index.html]: Should this link to the issue at https://github.com/blackbaud/skyux-layout/issues/94 instead of the project?

Comment (Blackbaud-ToddRoberts) [reply src/app/blog/index.html]: my thought is that people will want to see the progress of the project overall. I'm not planning on listing each item we are theming in coming months, just keep that item there.

**Pull request merged**